web/src/copy_and_paste.ts

```diff
@@ -221,8 +291,7 @@ export function copy_handler(): void {
     // Select div so that the browser will copy it
     // instead of copying the original selection
     select_div($div, selection);
-    // eslint-disable-next-line @typescript-eslint/no-deprecated
-    document.execCommand("copy");
+    await copy_selection_to_clipboard(selection);
```

Comment:
This particular call point, in particular, is just after `construct_copy_div`, which is a fancier version of constructing a range.